verify_colors_using_number.py

```diff
@@ -0,0 +1,16 @@
+from major_minor_colors_var import MAJOR_COLORS, MINOR_COLORS
+
+def get_color_from_pair_number(pair_number):
+  zero_based_pair_number = pair_number - 1
+  major_index = zero_based_pair_number // len(MINOR_COLORS)
+  if major_index >= len(MAJOR_COLORS):
+    raise Exception('Major index out of range')
+  minor_index = zero_based_pair_number % len(MINOR_COLORS)
+  if minor_index >= len(MINOR_COLORS):
+    raise Exception('Minor index out of range')
+  return MAJOR_COLORS[major_index], MINOR_COLORS[minor_index]
+
+def verify_colors_from_number(expected_major_color, expected_minor_color, pair_number):

```

Comment:
Declaration of test code is part of production code files. Production code needs to be kept separately from test code so that test code can be excluded during shipment of software to field.
Separation helps in maintenance and most importantly helps in testing production code independently and effectively.